.travis.yml

```diff
@@ -1,4 +1,5 @@
 sudo: false
+dist: trusty
```

Comment:
Necessary for Travis to be able to install OpenJDK 7 (otherwise it [fails](https://travis-ci.org/github/schibsted/spid-tech-docs/builds/680476711)).